packages/syft/src/syft/service/worker/worker_image.py

```diff
@@ -91,6 +101,10 @@ class SyftWorkerImage(SyftObject):
     image_hash: Optional[str]
     created_at: DateTime = DateTime.now()
     created_by: SyftVerifyKey
+    source_file: Optional[str]
+    full_tag_str: Optional[str]
```

Comment:
nitpick: This could just be `full_tag`? The type is already `str`.

Reply:
Done